app/command.go

```diff
@@ -176,24 +176,45 @@ func (a *App) ExecuteCommand(args *model.CommandArgs) (*model.CommandResponse, *
 
 	args.TriggerId = triggerId
 
-	if provider != nil {
-		if cmd := provider.GetCommand(a, args.T); cmd != nil {
-			response := provider.DoCommand(a, args, message)
-			return a.HandleCommandResponse(cmd, args, response, true)
-		}
+	if cmd, response := a.tryExecuteBuiltInCommand(args, trigger, message); cmd != nil && response != nil {
```

Comment:
2/5 I'd suggest the names `a.executeBuiltInCommand` and `a.executeCustomCommand` for even though `try...` is a little more intention-revealing. I think the code clarity would be improved by consistent naming altogether with ExecutePluginCommand.

Reply:
Will do

Reply:
Actually, `ExecutePluginCommand` isn't exposed publicly anywhere, so I'll just rename that to match